Makefile

```diff
@@ -95,6 +95,16 @@ dir: haskell-mode.info
 	$(INSTALL_INFO) --dir=$@ $<
 
 haskell-mode.tmp.texi: haskell-mode.texi
+	@sed -n -e '/@chapter/ s/@code{\(.*\)}/\1/' \
+                -e 's/@chapter \(.*\)$$/* \1::/p' \
+                -e 's/@unnumbered \(.*\)$$/* \1::/p' \
+               haskell-mode.texi > haskell-mode-menu-order.txt
+	@sed -e '1,/@menu/ d' \
+            -e '/end menu/,$$ d' \
+            haskell-mode.texi > haskell-mode-content-order.txt
+	diff -C 1 haskell-mode-menu-order.txt haskell-mode-content-order.txt
+	@rm haskell-mode-menu-order.txt haskell-mode-content-order.txt
+
```

Comment:
I may not have had enough coffee yet this morning, but my reading of this code suggests that the two `-order.txt` files are created and then immediately removed, such that the code has no effect. Please correct me if I'm wrong!